tools/serverpod_cli/lib/src/generator/dart/library_generators/util/model_filter_util.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:path/path.dart' as p;
+
+import 'package:serverpod_cli/src/analyzer/models/definitions.dart';
+import 'package:serverpod_cli/src/config/config.dart';
+
```

Comment:
Suggestion: For this file: Change the return type to Iterator where possible.

This allows users of the methods to combine them and still iterate the lists efficiently.